### ISSUE TITLE
Edit order language issues

### ISSUE THREAD

**Issue by BenjaminVerkleij** [opencart/opencart#3585]
Edit order language issues

If you run a multi language shop, editing an order can mess up the product language in the order.

To reproduce:
- Create a shop with 2 languages (lets say English and German).
- Make sure your test product uses both language product titles.
- Set the default language to English.
- Create an order on the frontend as German client with German language preference selected.
- Now as admin again, your frontend and backend language is english.
- Now edit the order on the backend as English admin.
- As soon as you hit the products tab, all product names jump to english language, Opencart doesn't respect the fact that the order was created with German language selected, even though that language preference is saved in the order.

The only work around as admin is to change you language preference on the fronted to German, and then edit the order again. 

Basically Opencart should respect the language preference in which the order is made, and not what the admin uses on the frontend.
